tox.ini

```
@@ -14,6 +14,8 @@ skip_install = true
 setenv =
     dev: H_EMBED_URL = {env:H_EMBED_URL:http://localhost:5000/embed.js}
     dev: CHECKMATE_URL = http://localhost:9099
+    dev: CHECKMATE_API_KEY = not_api_key
```

```suggestion
    dev: CHECKMATE_API_KEY = dev_api_key
```

It actually _is_ an API key :) It's just the one we use in dev

---

Updated it in all PRs